keps/sig-storage/3333-reconcile-default-storage-class/README.md

```diff
@@ -356,64 +356,41 @@ Unit tests will cover feature enablement/disablement.
 
 ### Rollout, Upgrade and Rollback Planning
 
-Will be provided when graduating to beta.
-
-<!--
-This section must be completed when targeting beta to a release.
--->
-
 ###### How can a rollout or rollback fail? Can it impact already running workloads?
 
-<!--
-Try to be as paranoid as possible - e.g., what if some components will restart
-mid-rollout?
-
-Be sure to consider highly-available clusters, where, for example,
-feature flags will be enabled on some API servers and not others during the
-rollout. Similarly, consider large clusters and how enablement/disablement
-will rollout across nodes.
--->
+In case the feature is not enabled during rollout due to a failure there
+should be no impact and the behavior of StorageClass assignment will not change.
+If the feature is rolled out partially, so it's enabled only on some API
+servers, there will be no impact on running workloads because the request will
+be  processed as if the feature is disabled - that means nothing happens and PVC
+will not be changed.
 
 ###### What specific metrics should inform a rollback?
 
-<!--
-What signals should users be paying attention to when the feature is young
-that might indicate a serious problem?
--->
+A KCM metric for failure counts called `retroactive_storageclass_fail` will
+indicate a problem with the feature.
 
 ###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
 
-<!--
-Describe manual testing that was done and the outcomes.
-Longer term, we may want to require automated upgrade/rollback tests, but we
-are missing a bunch of machinery and tooling and can't do that now.
--->
+Upgrade and rollback will be tested when the feature gate will change to beta.
```

Comment:
Add an item to Beta graduation criteria.

Reply:
Added.